pontoon/base/static/js/multiple_locale_selector.js

```diff
@@ -15,10 +15,12 @@ $(function() {
 
   // Choose locales
   $('body').on('click', '.multiple-locale-selector .locale.select li', function () {
-    var target = $(this).parents('.locale.select').siblings('.locale.select').find('ul'),
-        clone = $(this).remove();
+    var ls = $(this).parents('.locale.select'),
+        target = ls.siblings('.locale.select').find('ul'),
+        item = $(this).remove();
```

Comment:
These are changed for consistent code style with "Choose/remove all locales" lower in the file.